### ISSUE TITLE
PawerTrail --> GeoPath

### ISSUE THREAD

**Issue by andrixnet** [opencaching/opencaching-pl#613]
The entire code for GeoPaths should be rewritten to change all "PowerTrail" keywords to "GeoPath" due to potential copyright issues.

**Comment by kojoty** [comment]
it also affected okapi... there are also "powertrail" keywords...

**Comment by andrixnet** [comment]
@kojoty please review and integrate these changes as well: https://github.com/andrixnet/opencaching-pl/commit/22f2263586211fdbdeb746fe2fa2900633205afa

They are not yet the rework I described here https://github.com/opencaching/opencaching-pl/projects/2 but a correction of partially existing and non-working sizes.

**Comment by mzylowski** [comment]
@andrixnet Can you give some links or reasons why 'powerTrail' keyword is prohibited and why we have to do this transition ?

**Comment by andrixnet** [comment]
`powertrail` seems to have been used for some time by Groundspeak while they beta developed a new feature (today known as geotour). 
This has spawned a good number of bookmark lists on geocaching.com named "powertrail (XYZ)"
https://www.geocaching.com/bookmarks/view.aspx?guid=abe55ef7-55d4-4e49-9c35-b51a61c11e27
https://www.geocaching.com/bookmarks/view.aspx?guid=5816510f-d751-4a7e-84cb-055f6fb886ce
[etc]
Also, discussions and documents by others such as local organizations such as 
https://www.cachewiki.de/wiki/Liste:PowerTrails

Since then however the feature has been named `geotour` officially by Groundspeak ( see https://www.geocaching.com/play/geotours ), however the term `powertrail` has stuck.

In conclusion, in today's context, I would surmise: 

- `powertrail` is not "prohibited", but should be discouraged because of the potential confusion it may create, as refering to a geocaching.com feature or popular bookmark list names. 
- `geotour` most likely should be prohibited, because it is a Groundspeak feature and may raise copyright problems. 

@wrygiel already suggested careful choice of naming, with several alternate examples here: https://github.com/opencaching/okapi/pull/459#issuecomment-295473248

**Comment by andrixnet** [comment]
While I admit `geopath` was a term that quickly popped into my mind while working on the EN->RO translations of this feature, @wrygiel has a good point. 

Furthermore, given the spirit of **opencaching**, also having a feature called **openchecker**, the term `opentrail` sounds very good. ( see https://github.com/opencaching/okapi/pull/459#issuecomment-295474043 )

**Comment by harrieklomp** [comment]
When i read Powertrail i think at GC. It is a well known word for cachers. But it does not fit on OC :-)
Yes, i did like our Geopth but it had not the intention of a Powertrail.
When i read Opentrail it looks more like Powertrail and in my opinion it can be used as other word for Geopath.

Just my two cents :-)

**Comment by wrygiel** [comment]
> Also, discussions and documents by others such as local organizations such as 
> https://www.cachewiki.de/wiki/Liste:PowerTrails

This doesn't seem to be grounspeak-related page however.

> Since then however the feature has been named `geotour` officially by Groundspeak ( see https://www.geocaching.com/play/geotours ), however the term `powertrail` has stuck.

I wonder why this happened. My guess is, Groundspeak realized that "powertrails" were not "trails" at all. In our case, this is similar - our geopaths are not paths at all. They are "sets of caches in very different locations". So, Grounspeak decided to use the name "tour" instead, because it fits better. Problem is, it sounds crappy. "Powertrail" just sounds good.

Opentrail sounds good too. Even if most of these opentrails are not really trails. I still like it. And I guess Groundspeak's users also do (because they still tend to use "powertrail" instead of "geotour"?).
